src/libraries/System.Security.Permissions/src/CompatibilitySuppressions.xml

```diff
@@ -84,85 +76,57 @@
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
-    <Target>T:System.Security.Policy.Evidence</Target>
-    <Left>lib/net5.0/System.Security.Permissions.dll</Left>
-    <Right>lib/net5.0/System.Security.Permissions.dll</Right>
-    <IsBaselineSuppression>true</IsBaselineSuppression>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0001</DiagnosticId>
-    <Target>T:System.Security.Policy.EvidenceBase</Target>
+    <Target>T:System.Xaml.Permissions.XamlLoadPermission</Target>
     <Left>lib/net5.0/System.Security.Permissions.dll</Left>
-    <Right>lib/net5.0/System.Security.Permissions.dll</Right>
-    <IsBaselineSuppression>true</IsBaselineSuppression>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0001</DiagnosticId>
-    <Target>T:System.Security.Policy.Evidence</Target>
-    <Left>lib/netcoreapp3.0/System.Security.Permissions.dll</Left>
-    <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
-    <IsBaselineSuppression>true</IsBaselineSuppression>
-  </Suppression>
-  <Suppression>
-    <DiagnosticId>CP0001</DiagnosticId>
-    <Target>T:System.Security.Policy.EvidenceBase</Target>
-    <Left>lib/netcoreapp3.0/System.Security.Permissions.dll</Left>
     <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Security.Policy.Evidence</Target>
-    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Left>lib/net5.0/System.Security.Permissions.dll</Left>
     <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Security.Policy.EvidenceBase</Target>
-    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Left>lib/net5.0/System.Security.Permissions.dll</Left>
     <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Xaml.Permissions.XamlLoadPermission</Target>
-    <Left>ref/net5.0/System.Security.Permissions.dll</Left>
+    <Left>lib/netcoreapp3.0/System.Security.Permissions.dll</Left>
     <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Security.Policy.Evidence</Target>
-    <Left>ref/net5.0/System.Security.Permissions.dll</Left>
+    <Left>lib/netcoreapp3.0/System.Security.Permissions.dll</Left>
```

Comment:
Same question for the `lib/netcoreapp3.0` vs `lib/netstandard2.0`. If we were comparing `2.0` as the left and `3.0` to the right, I would understand it, but the other way around doesn't make much sense to me, am I missing something?

Reply:
we no longer contain any netcoreapp3.0 or lower assets so netstandard2.0 is the last fallback asset.